src/test/it/essentials.yml

```diff
@@ -0,0 +1,19 @@
+---
+ath:
+  useLocalSnapshots: false
+  athRevision: "dad333092159cb368efc2f9869572f0a05d255ac"
+  tests:
+    - "plugins.SshSlavesPluginTest"
+    - "core.SlaveTest"
+  categories:
+    - "org.jenkinsci.test.acceptance.junit.SmokeTest"
+pct:
+  useLocalSnapshots: false
+  jth:
+    version: 2.38
```

Comment:
Why is this even necessary? Just ensure that the downstream plugins are using a sufficiently new parent POM.

Part of the benefit from my PoV of the JTH split from core was that the specific version of JTH in use by a given plugin could be controlled by its POM, independently of core version. This was very important since there were sometimes incompatible changes, especially to HtmlUnit, and so the exact contents of `src/test/java/` sometimes needed to be synchronized with the JTH version.

Reply:
It is necessary to kick-off prototype without need to deliver fixes in many repos to ship it (note that releases would have been required for PCT if we use versioned releases).

Reply:
> deliver fixes in many repos

We cannot patch `ssh-slaves-plugin/pom.xml` and `command-launcher-plugin/pom.xml`?!